draft-ietf-lamps-kyber-certificates.md

```diff
@@ -74,24 +83,26 @@ and will use placeholders until these are released.]
 
 # Introduction
 
-The US NIST PQC Project has selected the Kyber algorithm
-as winner of their PQC Project {{PQCProj}}. This
-algorithm is a Key Encapsulation Mechanism(KEM). NIST has also defined object identifiers
-for these algorithms (TODO insert reference).
-
-This document specifies the use of the Kyber algorithm
-in X.509 public key certifiates, see {{!RFC5280}}.
-It also specifies private key encoding.
-An ASN.1 module is included for reference purposes.
-
-These certificates could be used as Issuers in CMS where the public key
-is used to encapsulate a shared secret used to derive a symmetric key
-used to encrypt content in CMS
-\[EDNOTE: Add reference draft-perret-prat-lamps-cms-pq-kem\].
-To be used in TLS, these certificates could only be used as end-entity
-identity certificates and would require significant updates to the
-protocol
-\[EDNOTE: Add reference draft-celi-wiggers-tls-authkem\].
+Kyber is a key-encapasulation mechanism (KEM). This document specifies
```

Comment:
If you keep this intro, just mention the word NIST PQ standardization effort. The rest is fine. Something like 
> [...] key-encapasulation mechanism (KEM) standardized in the US NIST PQC Project {{PQCProj}}